src/components/FileUpload/StyledFileUpload.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+const StyledFileUpload = styled.form`
+	margin-top: 15px;
+	text-align: center;
+	color: var(--primary-color);
+	z-index: 101;
+`;
+
+export { StyledFileUpload };
+
```

Comment:
Ich würde daraus zwei Components machen und nicht beides in einer Datei exportieren.